core/frontend/src/IModelApp.ts

```diff
@@ -100,8 +99,8 @@ export interface IModelAppOptions {
   accuDraw?: AccuDraw;
   /** If present, supplies the [[AccuSnap]] for this session. */
   accuSnap?: AccuSnap;
-  /** If present, supplies the [[I18N]] for this session. May be either an I18N instance or an I18NOptions used to create an I18N */
-  i18n?: I18N | I18NOptions;
+  /** If present, supplies the [[LocalizationClient]] for this session. */
+  localizationClient?: LocalizationClient;

```

Comment:
`localizationClient` is a bit long of a name... Maybe we just use `localization` or `localize`.

Then usage would be something like,

`IModelApp.localization.getLocalizedString()`

Reply:
Renamed to localization globally.